tests/run/trashcan.pyx

```diff
@@ -0,0 +1,144 @@
+# mode: run
+
+cimport cython
+
+
+# Count number of times an object was deallocated twice. This should remain 0.
+cdef int double_deallocations = 0
+def assert_no_double_deallocations():
+    assert double_deallocations == 0
```

Comment:
Could this reset the `double_deallocations` counter? Otherwise, if one test fails, all subsequent tests will also fail, and it will be unclear where to look first.